composer.json

```diff
@@ -137,18 +136,18 @@
     "twig/twig": ">=3.9.0 <3.14.0"
   },
   "require-dev": {
+    "chrome-php/chrome": "^1.4.0",
     "codeception/codeception": "^5.0.3",
     "codeception/module-symfony": "^3.1.0",
+    "composer/composer": "*",
     "ergebnis/phpstan-rules": "^2.0",
+    "gotenberg/gotenberg-php": "^1.1 || ^2.4",
     "phpstan/phpstan": "1.12.7",
     "phpstan/phpstan-symfony": "^1.3.5",
     "phpunit/phpunit": "^9.3",
-    "gotenberg/gotenberg-php": "^1.1 || ^2.4",
-    "composer/composer": "*",
-    "chrome-php/chrome": "^1.4.0",
-    "webmozarts/console-parallelization": "^2.1",
     "symfony/dotenv": "^6.4",
-    "symfony/runtime": "^6.4"
+    "symfony/runtime": "^6.4",
```

Comment:
It might be possible to use "^6.4 || ^7.1" for Symfony projects if you have avoided deprecations: https://symfony.com/doc/current/contributing/community/releases.html#backward-compatibility

Reply:
@NiklasBr  I reduced the deprecations, but unfortunately excluding those from Symfony. Unfortunately, it's tough to tackle those. The ones included in the task are the ones covered in the PR.

see: 
https://github.com/pimcore/pimcore/issues/17696#issuecomment-2422026149
https://github.com/pimcore/pimcore/pull/17712
https://github.com/pimcore/admin-ui-classic-bundle/pull/719